utils.ml

```diff
@@ -10,6 +10,15 @@ module Json = struct
 
   let get key assoc =
     Option.map snd (List.find_opt (fun (k, _) -> String.equal k key) assoc)
+
+  let mask_text text max_length =
```

Comment:
what is the purpose of `mask_text`? I can see you're using it to display the certificate and private key, but why?

Reply:
It just adds a bunch of asterisk. It's committed by error.